.photofinish.toml

```diff
@@ -202,3 +202,93 @@ files = [
 files = [
     "./test/fixtures/scenarios/host-details/9cd46919-5f19-59aa-993e-cf3736c71053_cloud_discovery_unknown.json",
 ]
+
+[host-vmdrbddev01-restore]
```

Comment:
I don't know if we should use any `drbd` thing for anything hehe
As they are candidate number one to be removed from our photofinish setup.
Maybe it will happen, but hey, just pointing it out